homeassistant/components/cover/services.yaml

```diff
@@ -6,27 +6,35 @@ open_cover:
   target:
     entity:
       domain: cover
+      supported_features:
+        - cover.CoverEntityFeature.OPEN
 
 close_cover:
   name: Close
   description: Close all or specified cover.
   target:
     entity:
       domain: cover
+      supported_features:
+        - cover.CoverEntityFeature.CLOSE
 
 toggle:
   name: Toggle
   description: Toggle a cover open/closed.
   target:
     entity:
       domain: cover
+      supported_features:
+        - 3 # cover.CoverEntityFeature.CLOSE | cover.CoverEntityFeature.OPEN
```

Comment:
This is quite error prone and can't be validated in tests.
We could instead update the `ENTITY_FILTER_SELECTOR_CONFIG_SCHEMA` to support a list of feature flags:
```yaml
  target:
    entity:
      domain: cover
      supported_features:
        - - cover.CoverEntityFeature.CLOSE
          - cover.CoverEntityFeature.OPEN
```

Reply:
That makes sense to me. Its a definitely a bit unwieldy as-is